.github/workflows/release.yaml

```diff
@@ -13,6 +13,7 @@ on:
       - poetry.lock
       - pyproject.toml
       - '.github/**.md'
+      - '.github/workflows/ci.yaml'
```

Comment:
Agents are pinned there.

Reply:
This should only stop the release workflow. E.g. we bump the agent, we run the tests on the PR but don't release when we merge to main, since the charm didn't really change. In principle, it is possible to sneak breaking changes in the ignored files, but only for the tests. If the charm is changed, the release workflow will still run.

Reply:
I am not 100% sure here, but let's try. It sounds safe.